app/lib/actions/remote_execution/run_ssh_job.rb

```diff
@@ -0,0 +1,36 @@
+module Actions
+  module RemoteExecution
+    class RunSSHJob < RunHostJob
```

Comment:
What about putting `RunSSHJob` and `RunProxySSHCommand` into `RemoteExecution::SSH` namespace, so that we have them grouped by provider.